nf_core/utils.py

```diff
@@ -75,15 +75,13 @@ def setup_requests_cachedir():
     """
     # Only import it if we need it
     import requests_cache
+    
 
-    cachedir = os.path.join(tempfile.gettempdir(), 'nfcore_cache')
+    cachedir = os.path.join(os.getenv("HOME"), '.nfcore_cache')
```

Comment:
What do you think about nesting this inside the `~/.nextflow` home directory? One fewer home directory.. (even if it is hidden)

Reply:
I would stick to the common application centric approach: hidden cache dirs and config dirs for each application. Nf-core is a different application then Nextflow.

Reply:
How about `.nfcore/cache/` instead then? Future-proofs us for adding more stuff later..?

Reply:
Would be a good idea I think 👍